ws-nextjs-app/pages/[service]/live/[id]/ShareButton/styles.ts

```diff
@@ -0,0 +1,50 @@
+import { Theme, css } from '@emotion/react';
+import pixelsToRem from '../../../../../../src/app/utilities/pixelsToRem';
+
+const styles = {
+  button: ({ palette, fontSizes, fontVariants, spacings, mq }: Theme) =>
+    css({
+      display: 'inline-flex',
+      alignItems: 'center',
+      color: palette.BLACK,
+      ...fontSizes.pica,
+      ...fontVariants.sansBold,
+      padding: `${spacings.FULL + spacings.HALF}rem`,
+      marginTop: `${spacings.FULL}rem`,
+      marginBottom: `${spacings.TRIPLE}rem`,
+      marginInlineStart: `${spacings.DOUBLE}rem`,
+      border: `${pixelsToRem(2)}rem solid ${palette.BLACK}`,
+      backgroundColor: 'transparent',
+      cursor: 'pointer',
+      '&:hover, &:focus-visible': {
+        color: palette.WHITE,
+        border: `${pixelsToRem(2)}rem solid ${palette.BRAND_BACKGROUND}`,
+        backgroundColor: palette.BRAND_BACKGROUND,
+        path: {
+          fill: palette.WHITE,
+          [mq.HIGH_CONTRAST]: {
```

Comment:
Is 'mq.HIGH_CONTRAST' forced-colours? https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors

Reply:
Yes it is https://github.com/bbc/simorgh/blob/19cf6ce2e63699fbe09f829f901c7a15acfe2d91/src/app/components/ThemeProvider/mediaQueries.ts#L46

Reply:
Great. Would be good if we could rename HIGH_CONTRAST to FORCE_COLOURS across the code base...